examples/multiple_node_versions/WORKSPACE

```diff
@@ -0,0 +1,41 @@
+workspace(name = "multiple_node_versions")
+
+# load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# http_archive(
+#     name = "rules_nodejs",
+#     sha256 = "a54a01b883beff66da057a1a9afc3be83de566d8ba78f5139f9b370026af5868",
+#     urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/4.4.0/rules_nodejs-core-4.4.0.tar.gz"],
+# )
+
+local_repository(
+    name = "rules_nodejs",
+    path = "../../../rules_nodejs",
+)
+
+local_repository(
+    name = "build_bazel_rules_nodejs",
+    path = "../../../rules_nodejs",
+)
+
+load("@build_bazel_rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains", "rules_nodejs_dependencies")
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
+
+# This just gives us bazel-skylib
+rules_nodejs_dependencies()
+
+# if changes are made to the order update the tests as the test_node_version_default uses this order
```

Comment:
I'd say "The order matters because Bazel will provide the first registered toolchain when a rule asks Bazel to select it"